### PR TITLE
chore: fix GitHub Actions etc.

### PR DESCRIPTION
**Pull request by kenjis**
Supersedes #46, #44

- update github/workflows/unused.yml
- bump actions/chache from 2 to 3
- bump actions/checkout from 2 to 3
- fix coding style

### CODE REVIEW

File: .github/workflows/unused.yml [hunk 1]
```diff
@@ -6,14 +6,14 @@ on:
       - develop
     paths:
       - '**.php'
-      - 'composer.**'
```
Comment (lonnie-vault) [inline .github/workflows/unused.yml:9]: I believe these need to stay as they are currently: 

https://docs.github.com/en/github-ae@latest/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Comment (kenjis) [reply .github/workflows/unused.yml:9]: Why? What's wrong?

```
*: Matches zero or more characters, but does not match the / character. For example, Octo* matches Octocat.
```

**Comment by kenjis** [comment]
All green!

File: .github/workflows/unused.yml (hunk 1 above)
Comment (lonnie-vault) [reply .github/workflows/unused.yml:9]: Maybe it doesn't need to match `/` for our needs. I believe @MGatner wrote this, maybe he can chime in.

Comment (kenjis) [reply .github/workflows/unused.yml:9]: The code comes from the latest devkit:
https://github.com/codeigniter4/devkit/blob/develop/src/Template/.github/workflows/unused.yml

Comment (MGatner) [reply .github/workflows/unused.yml:9]: Yes this was a mistake, it should be a single asterisk. Thanks for checking on my work though 🤗

**Review by MGatner — APPROVED** [review]
Looks good!

**Pull request merged**